src/algebra/continued_fractions/computation/basic.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2020 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import algebra.continued_fractions.basic
+import algebra.ordered_field
+import algebra.archimedean
+/-!
+# Computable Continued Fractions
+
+## Summary
+
+We formalise the standard computation of regular continued fractions for linear ordered floor
+fields. The algorithm is rather simple and connected to the Euclidean algorithm. It can be found,
+for example, on Wikipedia:
+https://en.wikipedia.org/wiki/Continued_fraction#Calculating_continued_fraction_representations.
```

Comment:
Could you explain the algorithm, here or in "Main definitions"? Your file should be as self-contained as possible, and moreover if you give references to the definitions and lemmas below when you describe the algorithm it will help a user locate the relevant parts he may need.